esileapclient/osc/v1/mdc/mdc_offer.py

```diff
@@ -157,6 +157,11 @@ def get_parser(self, prog_name):
             dest='resource_class',
             required=False,
             help="Specify offers' resource-class.")
+        parser.add_argument(
+            '--purpose',
+            dest='purpose',
+            required=False,
+            help="Show purposes of all the given leases")
```

Comment:
I took a closer look and realized that I was right the first time - this (and a few additions below) are not needed. We *do* want to add Purpose to lease outputs; we *don't* want to filter offers on purpose.